packages/radix-ui-themes/src/components/index.ts

```diff
@@ -149,7 +149,7 @@ export { Callout, CalloutRoot, CalloutIcon, CalloutText } from './callout';
 export * from './callout.props';
 export { Card } from './card';
 export * from './card.props';
-export { DataListRoot, DataListData, DataListItem, DataListLabel } from './data-list';
+export { DataListRoot, DataListValue, DataListItem, DataListLabel } from './data-list';
```

Comment:
Ultra nit:
```suggestion
export { DataListRoot, DataListItem, DataListLabel, DataListValue } from './data-list';
```

Reply:
Something satisfying in the parts order matching the alphabetical one now 🙂